bin/build.sh

```diff
@@ -1,43 +1,59 @@
 #!/bin/bash
 
+# Makes it so that all commands are displayed as they get executed
 set -x
 
 wca_url="https://worldcubeassociation.org/"
+stylesheet=$(realpath "./assets/style.css")
 
 # Find formal and legal Markdown files and build PDFs out of them.
 find 'documents' -name '*.md' | while read file; do
   echo "Processing $file, vanilla style"
 
   html_name="${file%.md}.html"
   pdf_name="${file%.md}.pdf"
+  
+  file_headline=$(head -n 1 "$file")
+  document_title=$(echo "$file_headline" | sed -E "s/#+\s*//")
+  
+  # Copy stylesheet to be used by the temporary HTML files
+  cp "$stylesheet" "$(realpath $(dirname "$file"))"
```

Comment:
I don't think you need to copy the file every time at all!
Can't you just pass the `$stylesheet` path directly to pandoc?

Reply:
Nope, I tried this for ages, but HTML just doesn't understand absolute file paths, so you have to give it a relative path next to itself, so I just copy the file everywhere. Either that or I'm not smart enough about how to force HTML to see an absolute path from root.